services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -1858,6 +1897,51 @@ public Builder snappingIncludeClosuresList(@Nullable List<Boolean> snappingClosu
       return this;
     }
 
+    /**
+     *  A semicolon-separated list of booleans affecting snapping of
+     *  waypoint locations to road segments.
+     *  If true, road segments statically closed, that is long-term, will be considered for snapping
+     *  (for example, road under construction).
+     *  If false, they will not be considered for snapping.
+     *  If provided, the number of snappingIncludeStaticClosures must be the same
+     *  as the number of waypoints.
+     *  However, you can skip a coordinate and show its position in the list with the ; separator.
+     *  If null, this parameter defaults to false.
+     * <p>
+     * Only available with the {@link DirectionsCriteria#PROFILE_DRIVING_TRAFFIC}.
+     *
+     * @param snappingStaticClosures a String representing a list of booleans
+     * @return this builder for chaining options together
+     */
+    @NonNull
+    public abstract Builder snappingIncludeStaticClosures(@Nullable String snappingStaticClosures);
+
+    /**
+     *  A list of booleans affecting snapping of waypoint locations to road segments.
+     *  If true, road segments statically closed, that is long-term, will be considered for snapping
+     *  (for example, road under construction).
+     *  If false, they will not be considered for snapping.
+     *  If provided, the number of snappingIncludeStaticClosures must be the same
+     *  as the number of waypoints.
+     *  However, you can skip a coordinate and show its position in the list with the null.
+     *  If null, this parameter defaults to false.
+     * <p>
+     * Only available with the {@link DirectionsCriteria#PROFILE_DRIVING_TRAFFIC}.
+     *
+     * @param snappingStaticClosures a list of booleans
+     * @return this builder for chaining options together
+     */
+    @NonNull
+    public Builder snappingIncludeStaticClosuresList(
+            @Nullable List<Boolean> snappingStaticClosures
+    ) {
+      String result = FormatUtils.join(";", snappingStaticClosures);
+      if (result != null) {
+        snappingIncludeStaticClosures(result);
+      }
+      return this;
+    }
```

Comment:
What does happen if you reset parameters from existing route options? 
```
RouteOptions.builder().
    // all all required parameters
    .build()
    .toBuilder()
    .snappingIncludeStaticClosuresList(null)
    .build()
```
or
```
RouteOptions.builder().
    // all all required parameters
    .build()
    .toBuilder()
    .snappingIncludeStaticClosuresList(emptyList())
    .build()
```
?
Do you think it makes sense to add tests for this?

Reply:
I added tests for null and empty list but without previous `toBuilder` invocation. I also added tests that nothing breaks after `toBuilder` + `build` (is this redundant?). I'm not sure tho why it would make sense to check resetting.

Reply:
resetting from a value to null looks suspicious in the code. It seems that in case of `null` `snappingIncludeStaticClosures` won't be called from `snappingIncludeStaticClosuresList ` and state won't change. As I see, coordinates work in the same way, so I'm not sure it's a bug. I think it would be nice to document that behaviour in the test. WDYT?

Reply:
Great catch! I'd say it's a bug. @LukasPaczos added this code, WDYT?

Reply:
That does look like a bug.